ocs_ci/utility/utils.py

```diff
@@ -3159,6 +3160,34 @@ def get_ocp_upgrade_history():
     return upgrade_history
 
 
+def get_attr_chain(obj, attr_chain):
+    """
+    Attempt to retrieve object attributes when uncertain about the existence of the attribute
+    or a different attribute in a given attribute chain. If the retrieval fails, None is returned.
+    The function can be used to retrieve a direct attribute, or a chain of attributes.
+    i.e. - obj.attr_a, obj_attr_a.sub_attr
+
+    Another example - trying to access "sub_attr_b" in object.attr.sub_attr_a.sub_attr_b -
+    get_attr_chain(object, "attr.sub_attr_a.sub_attr_b")
+
+    The function can be used to try and retrieve "sub_attribute_b" without an exception,
+    even in cases where "attr" or "sub_attr_a" might not exist.
+    In those cases, the function will return None.
+
+    Args:
+        obj: An object
+        attr_chain (str): A string containing one attribute or several sub-attributes
+                          separated by dots
```

Comment:
This should be indented only once and you can put the it on one line with the line bellow as well.